lib/assert.js

```diff
@@ -109,7 +109,7 @@ assert.ok = ok;
 // The equality assertion tests shallow, coercive equality with
 // ==.
 // assert.equal(actual, expected, message_opt);
-
+/* eslint-disable no-restricted-properties */
```

Comment:
I'd slightly prefer 8 `eslint-disable-next-line` calls. The more targeted and explicit the disabling, the better. But I don't oppose doing it the way you did it either.